api/sql/schema/750-rule.rule.remove.sql

```diff
@@ -6,64 +6,6 @@ DECLARE @userId BIGINT = (SELECT [auth.actorId] FROM @meta)
 BEGIN TRY
 
     BEGIN TRANSACTION
-        DELETE x
-        FROM
-            [rule].limit x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE x
-        FROM
-            [rule].conditionActor x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE x
-        FROM
-            [rule].conditionItem x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE x
-        FROM
-            [rule].conditionProperty x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
-        DELETE
-            x
-        FROM
-            [rule].splitRange x
-        JOIN
-            [rule].splitName s ON s.splitNameId = x.splitNameId
-        JOIN
-            @conditionId item ON s.conditionId = item.value
-        DELETE
-            x
-        FROM
-            [rule].splitAnalytic x
-        JOIN
-            [rule].splitAssignment y ON y.splitAssignmentId = x.splitAssignmentId
-        JOIN
-            [rule].splitName s ON s.splitNameId = y.splitNameId
-        JOIN
-            @conditionId item ON s.conditionId = item.value
-
-        DELETE
-            x
-        FROM
-            [rule].splitAssignment x
-        JOIN
-            [rule].splitName s ON s.splitNameId = x.splitNameId
-        JOIN
-            @conditionId item ON s.conditionId = item.value
-        DELETE
-            x
-        FROM
-            [rule].splitName x
-        JOIN
-            @conditionId item ON x.conditionId = item.value
-
```

Comment:
Deleting the above is not needed. The splitNameId column is returned by rule.decision.fetch and can be stored in other tables, to it is not a good idea to delete from splitName.